app/animals/page.tsx

```diff
@@ -7,8 +7,6 @@ export const metadata = {
   description: 'My favorite animals',
 };
 
-export const dynamic = 'force-dynamic';
```

Comment:
When calling a Dynamic Function as headers() in sql client there is no need to write this line since [Next.js will assume this is a Dynamic Rendered page](https://nextjs.org/docs/app/building-your-application/rendering/static-and-dynamic-rendering#dynamic-rendering)